com.unity.shadergraph/Editor/Drawing/Views/StickyNote.cs

```diff
@@ -1,5 +1,9 @@
+using System;
```

Comment:
Check these includes, some are probably not necessary.

Reply:
Removed the unneeded ones 👍